packages/client/components/ActivityLibrary/ActivityLibrary.tsx

```diff
@@ -23,10 +44,32 @@ export const ActivityLibrary = (props: Props) => {
   const data = usePreloadedQuery<ActivityLibraryQuery>(query, queryRef, {
     UNSTABLE_renderPolicy: 'full'
   })
+  const {viewer} = data
+  const {featureFlags, availableTemplates} = viewer
+
+  const templates = [
+    {id: 'action', type: 'action', name: 'Check-in', team: {name: 'Parabol'}},
```

Comment:
I'm thinking since we'll probably need to do some special-casing on the client-side for standups+check-ins, it _might_ be better to add these on the client-side - i.e. the server gives the client all the meeting _templates_, and it's up to the client to use those templates to show the available activities.